src/commands/command-docs.json

```diff
@@ -18,6 +18,188 @@
         "command_tips": [
             "NONDETERMINISTIC_OUTPUT_ORDER"
         ],
+        "reply_scema": {
+            "description": "A map where each key is a command name, and each value",
+            "type": "object",
+            "patternProperties": {
```

Comment:
is this really necessary? all entries in COMMAND DOCS have the same properties so why not just use `properties`?

Reply:
oh ok i got it (because otherwise the schema would look for `summary` and will actually get `PINGz or whatever)